ui/src/pages/session-details/playback/activity-plot/activity-plot.tsx

```diff
@@ -69,14 +90,30 @@ export const ActivityPlot = ({
               zeroline: false,
               rangemode: 'nonnegative',
               fixedrange: true,
+              range: [yAxisMin, yAxisMax],
+              side: 'left',
             },
-            xaxis: {
-              showline: true,
+            yaxis2: {
               showgrid: false,
               showticklabels: false,
               zeroline: false,
+              rangemode: 'nonnegative',
+              fixedrange: true,
+              range: [0, session.detectionsMaxCount],
+              side: 'right',
+              overlaying: 'y',
+            },
+            xaxis: {
+              showline: false,
+              showgrid: true,
+              griddash: 'dot',
+              gridwidth: 1,
+              gridcolor: gridLineColor,
+              showticklabels: false,
+              zeroline: false,
               fixedrange: true,
-              range: [session.startDate, session.endDate],
+              range: [new Date(session.startDate), new Date(session.endDate)],
```

Comment:
`session.startDate` is already a date, any reason we are creating a new date? :)

Reply:
You are right, I was experimenting with date formats was trying different approaches. Reverting!